css-canvas-medical/styles.css

```diff
@@ -0,0 +1,186 @@
+.container {
+  max-width: 1300px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+  padding-top: 1rem;
+  padding-bottom: 2rem;
+}
+
+.header {
+  position: relative;
+  top: 1rem;
+  border-bottom: solid;
+  border-bottom-color: whitesmoke;
+}
+
+.nav {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  position: absolute;
+  top: 0.5rem;
+  left: 43.7rem;
```

Comment:
remember when using absolute positioning, you dont have to push from the original location. right should work well here.